app/models/goal.py

```diff
@@ -3,3 +3,17 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
```

Comment:
A cool thing we can do is make sure that data with required fields aren't created with NULL values with the argument `nullable=False`. This can help if no conditional in the backend checks for empty fields. Sometimes things can slip through the backend, but this argument will add another layer of defense!